python/paddle/v2/framework/tests/test_sgd_op.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+import numpy
+from op_test_util import OpTestMeta
+
+
+class TestSGD(unittest.TestCase):
+    __metaclass__ = OpTestMeta
+
+    def setUp(self):
+        self.type = "sgd"
+        self.param = numpy.random.random((342, 345)).astype("float32")
+        self.grad = numpy.random.random((342, 345)).astype("float32")
+        self.learning_rate = 0.1
+        self.param_out = self.param - self.learning_rate * self.grad
+
```

Comment:
By using current op unit test framework, the developer only needs to `setUp` this variable, the test method is injected by `OpTestMeta`.